packages/isomorphic/README.md

```diff
@@ -0,0 +1,100 @@
+# @xrplf/isomorphic
```

Comment:
Is there a reason there isn't an `index.ts` to import all of the functions if you're not worried about tree-shaking and you just want something easy to use?

Reply:
We want to encourage using entry points. Many projects do this already. This also prevents future contributors from accidentally importing the whole thing.